assets/templates/signup.html

```diff
@@ -37,42 +37,46 @@
       </header><!-- header -->
       <h5>Regístrate</h5>
-      <p class="content-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p><!-- Title -->
+      <p class="content-subtitle">Crea una cuenta GRATIS hasta 500 Suscriptores.</p><!-- title and subtitle -->
       <form class="signup-form">
         <fieldset>
           <legend>Creación de cuenta</legend>
-          <ul>
-            <li class="holder-small">
-              <label for="name">Nombre:</label>
-              <input type="name" name="lastname" placeholder="Nombre">
+          <ul class="holder-group">
+            <li class="holder-size1of1">
+              <ul class="holder-group holder-col">
+                <li class="holder-size1of2">
+                  <label for="name">Nombre:</label>
+                  <input type="name" name="lastname" placeholder="Nombre">
+                </li>
+                <li class="holder-size1of2">
+                  <label for="lastname">Apellido:</label>
+                  <input type="lastname" name="name" placeholder="Apellido">
+                </li>
+              </ul>
             </li>
-            <li class="holder-small">
-              <label for="lastname">Apellido:</label>
-              <input type="lastname" name="name" placeholder="Apellido">
-            </li>
-            <li>
+            <li class="holder-size1of1">
               <label for="phone">Teléfono:</label>
               <input type="tel" name="phone" placeholder="9 11 2345-6789">
             </li>
-            <li>
+            <li class="holder-size1of1">
               <label for="email">Email:</label>
               <input type="email" name="email" placeholder="Tu Email será tu nombre de usuario">
             </li>
-            <li>
+            <li class="holder-size1of1">
               <label for="password">Contraseña:</label>
               <input type="password" name="password" placeholder="Escribe tu clave secreta">
             </li>
           </ul>
-        </fieldset>
-        <fieldset class="holder-policy">
+        </fieldset><!-- personal information, user and password -->
+        <fieldset>
           <legend>Politica de privacidad y promociones</legend>
-          <ul class="content-checkbox">
-            <li class="checkbox">
+          <ul class="checkbox-group">
+            <li class="holder-checkbox">
               <input type="checkbox" name="acepto_politicas">
               <span class="checkmark"></span>
               <label for="acepto_politicas">Acepto la <a class="link-green" href="#">Política de Privacidad</a> de Doppler.</label>
             </li>
-            <li class="checkbox">
+            <li class="holder-checkbox">
               <input type="checkbox" name="acepto_promociones">
               <span class="checkmark"></span>
               <label for="acepto_promociones">Acepto las promociones de Doppler y sus aliados.</label>
```

Comment:
Label baseline should be the bottom of the checkbox, so it can be aligned properly.

![image](https://user-images.githubusercontent.com/3520460/54615431-dbf0df00-4a3c-11e9-8c76-9a80404ec09a.png)

Reply:
It is not possible, because the input was replaced by a span that contains the correct design. What you can do is put a height so that it does not align itself.